contracts/package.json

```diff
@@ -13,8 +13,7 @@
     "@openzeppelin/upgrades-core": "^1.34.1",
     "chalk": "^5.3.0",
     "ethers": "^6.13.1",
-    "openzeppelin-foundry-upgrades": "github:OpenZeppelin/openzeppelin-foundry-upgrades",
-    "solidity-stringutils": "github:Arachnid/solidity-stringutils"
```

Comment:
use `solidity-stringutils` for upgradable contract

Reply:
<img width="992" alt="image" src="https://github.com/zuni-lab/zuni-donator/assets/69207570/c78afbd6-0a9e-4264-a9c2-b6758321c3b4">